pythonscript/_godot_script.pxi

```diff
@@ -222,5 +222,6 @@ cdef api void pythonscript_script_finish(
 ) with gil:
     cdef object cls = <object>p_data
     if get_pythonscript_verbose():
-        print(f"Destroying python script {cls.__name__}")
+        # Using print here will cause a crash on editor/game shutdown
+        sys.__stdout__.write(f"Destroying python script {cls.__name__}\n")
```

Comment:
I'm a bit scared by this (this would mean we're never safe to use `print` anywhere in the codebase...)

For instance if the user starts a thread in Python, it won't stop until `godot_gdnative_terminate` gets called, hence using `print` from it could end up in segfault...

Can't we do something in the `godot_gdnative_terminate` to revert the stdout/stderr configuration to default system instead ?

Reply:
Yeah, you're right, we should detect shutdown and revert stdout/stdin modifications. I'll work on that.